frontend/src/game/card/CardBase.jsx

```diff
@@ -130,8 +130,26 @@ const CardBaseText = ({ name, manaCost, type = "", rarity = "", power = "", toug
 
     <div className="body">
       {
-        text &&
-          <div className="text">{text}</div>
+        text && (
+          <div className="text">
+            { 
+              text
+                .split('\n')
+                .map(line => {
+                  const bracketSection = line.match(/\([^\)]+\)/g)
+                  if (!bracketSection) return line
+
+                  const [before, after] = line.split(bracketSection[0])
+                  return [
+                    before,
+                    <span className='bracket'>{bracketSection[0]}</span>,
```

Comment:
NOTE - this assume there will only be one bracketed helper text section _per line_. I think this is a reasonable assumption to make for 99% of cards and massively increases the readability of card with a lot of helper text